packages/manager/test/lib-mockSlice.test.ts

```diff
@@ -31,13 +28,13 @@ describe("mockSlice", () => {
 		};
 		const geoR = GeoPointContent.decode(geo.key);
 
-		expect(isRight(geoR)).toBeTruthy();
+		expect(isRight(geoR)).toBeFalsy();
```

Comment:
This should be truthy no? You likely need to uncomment line 25

Left as is (as I understand it) it defeats the purpose of the test (which also has no reason to be a test in first place here, we shouldn't have to test types-internal here)